gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.java

```diff
@@ -70,7 +70,11 @@
             // Set DNS cache TTL to 20s to account for systems such as RDS and other
             // AWS-managed systems that modify DNS records on failover.
             "-Dsun.net.inetaddr.ttl=20",
-            "-XX:NativeMemoryTracking=summary");
+            "-XX:NativeMemoryTracking=summary",
+            // Increase default JFR stack depth beyond the default (conservative) 64 frames.
+            // This can be overridden by user-provided options.
+            // See sls-packaging#1230
+            "-XX:FlightRecorderOptions=stackdepth=256");
```

Comment:
I considered a separate optional integer property to opt out of the setting or provide a different value, but I don't think it will be common enough to modify to be worthwhile.

Reply:
Agreed

Reply:
I guess the small? downside is that capturing a JFR might have a slightly larger perf overhead? But going from 64 to 256 sounds reasonable.